Containerfile.base

```diff
@@ -0,0 +1,32 @@
+# Contains embedding model and dependencies needed to generate database
+
+ARG EMBEDDING_MODEL=sentence-transformers/all-mpnet-base-v2
```

Comment:
We could encapsulate the embedding model into the base image a bit better by storing its name in a file and then extracting it from that file when we build a derived image. This way we won't have to specify the name of the embedding model in > 1 place.

Reply:
We could just turn it into an `ENV` that would propagate it to other stages.

Reply:
Yep, that'd be great.

Reply:
Done. The embedding model is now defined under `ENV`.